vmm/src/memory_manager.rs

```diff
@@ -606,7 +606,7 @@ impl MemoryManager {
                             false,
                             config.shared,
                             config.hugepages,
-                            None,
+                            zone.host_numa_node,
```

Comment:
Looks great! Could you please add two additional patches to fix `config.shared` and `config.hugepages` as well?
They should be `zone.shared` and `zone.hugepages` instead.

Reply:
Done

Reply:
Thanks!